tests/client/amo/components/TestSearchSort.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import {
+  Simulate,
+  findRenderedComponentWithType,
+  renderIntoDocument,
+} from 'react-addons-test-utils';
+import { Provider } from 'react-redux';
+
+import createStore from 'amo/store';
+import I18nProvider from 'core/i18n/Provider';
+import SearchSort from 'amo/components/SearchSort';
+import { getFakeI18nInst } from 'tests/client/helpers';
+
+
+function render(props) {
+  const initialState = { api: { clientApp: 'android', lang: 'en-GB' } };
+
+  return findRenderedComponentWithType(renderIntoDocument(
+    <Provider store={createStore(initialState)}>
+      <I18nProvider i18n={getFakeI18nInst()}>
+        <SearchSort {...props} />
+      </I18nProvider>
+    </Provider>
+  ), SearchSort);
+}
+
+describe('<SearchSortBase />', () => {
+  it('toggles the search sort options when clicking "sort" link', () => {
+    const root = render({ filters: { query: 'test' }, pathname: '/search/' });
+    const rootNode = findDOMNode(root);
+
+    assert.notInclude(rootNode.className, 'SearchSort--visible');
+
+    Simulate.click(rootNode.querySelector('.SearchSort-toggle'));
+
+    assert.include(rootNode.className, 'SearchSort--visible');
+  });
+
+  it('blurs the search sort toggle when closing the sort options', () => {
+    const searchToggle = { blur: sinon.stub() };
+    const root = render({
+      _searchToggle: searchToggle,
```

Comment:
If you use `event.currentTarget` in `onClick` then you should be able to accomplish this without `_searchToggle`.

```js
// May need to set preventDefault, too.
Simulate.click(root.searchToggle, { currentTarget: searchToggle });
```

Alternatively, rather than passing in `_searchToggle` you could check that it is set to the right element then stub it.

```js
assert.ok(root.searchToggle.classList.contains('SearchSort-toggle'));
sinon.stub(root, 'searchToggle', searchToggle);